src/pybind/mgr/cephadm/ssl_cert_utils.py

```diff
@@ -42,6 +43,7 @@ def generate_root_cert(
         root_builder = root_builder.public_key(root_public_key)
 
         san_list: List[x509.GeneralName] = []
+        san_list.append(x509.DNSName(f'fsid-{self.cluster_fsid}'))  # Add FSID as a DNSName in SAN
```

Comment:
I think the comment in this case is just describing what the code is already doing, so I think we can get rid of it :+1: